editor/debugger/editor_visual_profiler.cpp

```diff
@@ -453,44 +453,57 @@ void EditorVisualProfiler::_graph_tex_draw() {
 	int font_size = get_theme_font_size(SceneStringName(font_size), SNAME("Label"));
 	const Color color = get_theme_color(SceneStringName(font_color), EditorStringName(Editor));
 
+	const int half_width = graph->get_size().x / 2;
+	// Draw a line in the middle to separate the CPU graph from the GPU graph.
+	graph->draw_line(Vector2(half_width, 0), Vector2(half_width, graph->get_size().y), color * Color(1, 1, 1, 0.3), Math::round(EDSCALE));
```

Comment:
`EDSCALE` can be less than one, so can be rounded to zero if it's too small (UI clamps it to `0.5`), and seems like `draw_line` will draw nothing if width is zero (bug?).

Reply:
`Math::round(EDSCALE)` is a pattern we use a lot currently, and I don't think anyone is using an editor scale below 0.5. Moreover, `0.5` rounds to `1.0` anyway since `round()` always rounds away from 0. 0.75 is the smallest preset available either way, and the smallest value I expect to be practically usable (it was designed for 1366x768 displays).